pkg/controllers/dynakube/extension/otel/statefulset.go

```diff
@@ -18,30 +19,37 @@ import (
 )
 
 const (
-	serviceAccountName              = "dynatrace-extensions-collector"
-	containerName                   = "collector"
-	tokenSecretKey                  = "otelc.token"
-	caCertsVolumeName               = "cacerts"
-	defaultImageRepo                = "public.ecr.aws/dynatrace/dynatrace-otel-collector"
-	defaultImageTag                 = "0.7.0"
-	defaultOLTPgrpcPort             = "10001"
-	defaultOLTPhttpPort             = "10002"
-	defaultPodNamePrefix            = "extensions-collector"
-	defaultReplicas                 = 1
-	envShards                       = "SHARDS"
-	envShardId                      = "SHARD_ID"
-	envPodNamePrefix                = "POD_NAME_PREFIX"
-	envPodName                      = "POD_NAME"
-	envOTLPgrpcPort                 = "OTLP_GRPC_PORT"
-	envOTLPhttpPort                 = "OTLP_HTTP_PORT"
-	envOTLPtoken                    = "OTLP_TOKEN"
-	envTrustedCAs                   = "TRUSTED_CAS"
+	serviceAccountName   = "dynatrace-extensions-collector"
+	containerName        = "collector"
+	tokenSecretKey       = "otelc.token"
+	caCertsVolumeName    = "cacerts"
+	defaultImageRepo     = "public.ecr.aws/dynatrace/dynatrace-otel-collector"
+	defaultImageTag      = "0.12.0"
+	defaultOLTPgrpcPort  = "10001"
+	defaultOLTPhttpPort  = "10002"
+	defaultPodNamePrefix = "extensions-collector"
+	defaultReplicas      = 1
+	envShards            = "SHARDS"
+	envShardId           = "SHARD_ID"
+	envPodNamePrefix     = "POD_NAME_PREFIX"
+	envPodName           = "POD_NAME"
+	envOTLPgrpcPort      = "OTLP_GRPC_PORT"
+	envOTLPhttpPort      = "OTLP_HTTP_PORT"
+	envOTLPtoken         = "OTLP_TOKEN"
+	envTrustedCAs        = "TRUSTED_CAS"
+	// certDirEnv is the environment variable which identifies which directory
+	// to check for SSL certificate files. If set this overrides the system default.
+	// It is a colon separated list of directories.
+	// See https://www.openssl.org/docs/man1.0.2/man1/c_rehash.html.
+	envCertDir                      = "SSL_CERT_DIR"
```

Comment:
(this complaint stems from the "original" structure of the code, and not your change specifically, and fixing it would make you change look even more complicated than it actually is, so might make sense to do it separately, up to you)

Reply:
@0sewa0 you are absolutely right, i did similar grouping in EEC package, so tried to apply it here as well.
pleas take a look on 97ca94c6

Reply:
here you can see similar part. https://github.com/dynatrace/dynatrace-operator/blob/912eddcfb50811e4a7ed489036b766e4875d447b/pkg/controllers/dynakube/extension/eec/statefulset.go#L28-L67.